### PR TITLE
Checking alert properties presence

### PR DESCRIPTION
**Pull request by tsebastiani**

### CODE REVIEW

File: kraken/prometheus/client.py [hunk 1]
```diff
@@ -74,5 +79,5 @@ def critical_alerts(prom_cli: KrknPrometheus,
     if post_critical_alerts_count > 0:
         firing_alerts = True
 
-    if not firing_alerts:
+    if not firing_alerts:-
```
Comment (paigerube14) [inline kraken/prometheus/client.py:82]: typo?

Comment (tsebastiani) [reply kraken/prometheus/client.py:82]: yeah....

Comment (tsebastiani) [reply kraken/prometheus/client.py:82]: apparently it was already there

Comment (tsebastiani) [reply kraken/prometheus/client.py:82]: ah , no, my fault!

**Pull request merged**